scripts/update-copy-mod-doc-exceptions.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 
-# This is a little wrapper around the python script
-# lint-copy-mod-doc.py
+# This script completely regenerates the scripts/copy-mod-doc-exceptions.txt file
+# Typically this should only be run by automation.
+# Human contributors shouldn't need to run this unless they are making changes to the linting script
 
-find src archive -name '*.lean' | xargs ./scripts/lint-copy-mod-doc.py | sort > scripts/copy-mod-doc-exceptions.txt
+# Replace the current exceptions file with an empty file
+> scripts/copy-mod-doc-exceptions.txt
```

Comment:
This is already implicitly happening on the line below, with the `> scripts/copy-mod-doc-exceptions.txt`.